data/tlg0064/tlg001/__cts__.xml

```diff
@@ -0,0 +1,9 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0064" xml:lang="grc" urn="urn:cts:greekLit:tlg0064.tlg001">
+  <ti:title xml:lang="lat">Periplus Hannonis</ti:title>
+  <ti:edition urn="urn:cts:greekLit:tlg0064.tlg001.1st1K-grc1" workUrn="urn:cts:greekLit:tlg0064.tlg001">
+    <ti:label xml:lang="lat">
+      Periplus Hannonis, Geographi graeci minores Volumen Primum
+    </ti:label>
+    <ti:description xml:lang="eng">Hanno, creator; Müller, Karl, of Paris, editor</ti:description>
```

Comment:
We usually put the publisher and year of publication here as well. This is the metadata that shows up on the Github page, so it is good to have a bit more detail